src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let data;
+
```

Comment:
* [ ] both `numberOne` and `numberTwo` should be wrapped in big.js, here is how you could handle it.
`const num1 = Big(numberOne)`
`const num2 = Big(numberTwo)` 
In your subsequent code you could the new declared variable for the operation.

Reply:
Done